src/components/toggle-button/index.ts

```diff
@@ -0,0 +1 @@
+export { default as AssistantDepartureToggleButton } from './assistant-departure-toggle-button';
```

Comment:
Is this a toggle button, or is it more like a tab link?

Reply:
It is more like a tab link, currently used to switch between two URL´s. Do you have a suggestion to a more precise component name?

Reply:
I usually try to end "actions" (so things happening locally on site that doesn't trigger navigational URL change) with "Button" and navigational changes (links that trigger URL change) with "Link". Implying what kind of change they represent, and also specifying how they should be used.